Assets/Scripts/InventoryScripts/JournalInventoryScript.cs

```diff
@@ -1,147 +1,28 @@
-                {
-                    case 0:
-                        DeceitfulPages[0] = _page;
-                        pageItemPlace[8].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[8].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                    case 1:
-                        DeceitfulPages[1] = _page;
-                        pageItemPlace[9].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[9].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                    case 2:
-                        DeceitfulPages[2] = _page;
-                        pageItemPlace[10].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[10].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                    case 3:
-                        DeceitfulPages[3] = _page;
-                        pageItemPlace[11].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[11].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                }
-
-                break;
-            case 4:
-                switch (dialogueScr._pageClass)
-                {
-                    case 0:
-                        UnfortunatePages[0] = _page;
-                        pageItemPlace[12].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[12].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                    case 1:
-                        UnfortunatePages[1] = _page;
-                        pageItemPlace[13].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[13].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                    case 2:
-                        UnfortunatePages[2] = _page;
-                        pageItemPlace[14].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[14].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                    case 3:
-                        UnfortunatePages[3] = _page;
-                        pageItemPlace[15].image.overrideSprite = _page.GetComponent<SpriteRenderer>().sprite;
-                        pageItemPlace[15].onClick.AddListener(() => dialogueScr.Interact(_thePlayer));
-                        break;
-                }
-
-                break;
-        }
-    }
-
-    private void CheckPage()
-    {
+        Sprite spriteRdr = _page.GetComponent<SpriteRenderer>().sprite;
```

Comment:
Contractions are sort of frowned upon, especially if it is not super clear. `spriteRdr` sounds like a `SpriteRenderer`, but this is actually a `Sprite`.